embd.go

```diff
@@ -30,6 +30,7 @@ import (
 	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"regexp"
```

Comment:
What do you think it can look better at the top of output like
```
$ ./embd-go
embd-go version 1.1 (2016-01-04) https://github.com/akavel/embd-go
USAGE: embd-go [FLAGS] PATH...
  -o string
    	Path to generated file. (default "embd/data.go")
  -p string
    	Package that the generated file should be in. (default "embd")
```
or at the bottom like this
```
$ ./embd-go
USAGE: embd-go [FLAGS] PATH...
  -o string
    	Path to generated file. (default "embd/data.go")
  -p string
    	Package that the generated file should be in. (default "embd")
embd-go version 1.1 (2016-01-04) https://github.com/akavel/embd-go
```